css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
I've never used LESS which is why I went with the CDN option, am imagining it's very similar to SASS, but decided to do what was comfortable at the time. But I think once I'm more familiar I will go straight to LESS as well.